alf/utils/value_ops.py

```diff
@@ -61,12 +69,16 @@ def action_importance_ratio(training_info: TrainingInfo, action_spec,
     if check_numerics:
         importance_ratio = tf.debugging.check_numerics(importance_ratio,
                                                        'importance_ratio')
-    importance_ratio_clipped = tf.clip_by_value(importance_ratio,
-                                                1 - importance_ratio_clipping,
-                                                1 + importance_ratio_clipping)
+
+    if calling_class_name == 'PPOLoss':
+        importance_ratio_clipped = tf.clip_by_value(
+            importance_ratio, 1 - importance_ratio_clipping,
+            1 + importance_ratio_clipping)
+    else:  #if name_scope == VTraceLoss.__class__.__name__:
+        raise Exception('Unsupported calling class: ' + calling_class_name)
 
     if debug_summaries and common.should_record_summaries():
-        with tf.name_scope(name_scope):
+        with tf.name_scope(scope):
```

Comment:
with scope:

Reply:
Nice, I didn't know this works!  Addressed all comments, please take another look.